src/test/java/com/example/java/maven/ticketVendingMachine/TicketVendingMachineTest.java

```diff
@@ -0,0 +1,250 @@
+package com.example.java.maven.ticketVendingMachine;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+public class TicketVendingMachineTest {
```

Comment:
how did you feel writing tests for `TicketVendingMachine` class? :D was it easy? :D

Reply:
Not great, not terrible :D